### PR TITLE
Add bulk_apply_bypass_acls

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description

Add bulk_apply_bypass_acls

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* New API call /api/v1/nodes/bulk_apply_bypass_acls

### CODE REVIEW

File: t/unittest/UnifiedApi/Controller/Nodes.t [hunk 1]
```diff
@@ -115,6 +115,18 @@ $t->post_ok('/api/v1/nodes/bulk_apply_bypass_vlan' => json => { bypass_vlan => u
...
+$t->post_ok('/api/v1/nodes/bulk_apply_bypass_acls' => json => { bypass_acls => "permit ip any any",  items => [$mac] })
+  ->status_is(200)
+  ->json_is('/items/0/mac', $mac)
+  ->json_is('/items/0/status', 'success');
+
+$t->get_ok("/api/v1/node/$mac")
+  ->status_is(200)
+  ->json_is('/item/bypass_acls', 'permit ip any any');
+
+$t->post_ok('/api/v1/nodes/bulk_apply_bypass_acls' => json => { bypass_acls => "permit ip any",  items => [$mac] })
```
Comment (julsemaan) [inline t/unittest/UnifiedApi/Controller/Nodes.t:145]: Add a test to set it to empty

Comment (jrouzierinverse) [reply t/unittest/UnifiedApi/Controller/Nodes.t:145]: Done
